js/dynamicHtml.js

```diff
@@ -0,0 +1,90 @@
+const speakers = [
+  {
+    image: 'image1',
+    name: 'Ashley J. Blackman',
+    title: 'Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    brief: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.",
+  },
+
+  {
+    image: 'image2',
+    name: 'Laurie A. Johnson',
+    title: 'Art Lover',
+    brief: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.",
+  },
+
+  {
+    image: 'image3',
+    name: 'Helen J. Obrien',
+    title: 'Musicales Artist',
+    brief: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.",
+  },
+
+  {
+    image: 'image4',
+    name: 'George Hernandez',
+    title: 'President of Young Pirates, Europe',
+    brief: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.",
+  },
+
+  {
+    image: 'image5',
+    name: 'Brad McAvoy',
+    title: 'Director of Art Center, Nabi',
+    brief: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.",
+  },
+
+  {
+    image: 'image6',
+    name: 'Yolanda Lee',
+    title: 'Executive Director at Wikimedia Foundation',
+    brief: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.",
+  },
+];
+
+const imagePath = '../images/';
```

Comment:
- [ ] The images are not displayed because you write the wrong path, try `const imagePath = './images/';` and It will work properly. Also, consider making sure to fix these changes also in a live demo.